Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM unocha/debian-snap-base:10-buster-chrome81-node12-202001-01 as builder
+FROM unocha/debian-snap-base:10-buster-chrome88-node12-202010-01 as builder
```

Comment:
according to our new process, Chrome gets downloaded when the image is built on dockerhub, so I think it's safe to take the chrome version out of these tags.

Reply:
I saw that but didn't wanted to break the rule by myself. Happy to make an alias for this one and we can use the simplified tag next time. One question though: why do we install the chrome unstable only to install the newer chrome over it? Since these two chrome versions are part of different layers, they will both be part of the image size (even though on the last image layer the unstable gets erased). Wouldn't be better if we skip the unstable chrome version from the base image?

Reply:
Maybe it is being used by the other snap image? (used by @berliner)

Reply:
Well, no. Although it does use the same base image, the same mo is at work. Chrome stable is being added on top of the unstable one. This seems strange. requesting opinions from @rupl @cafuego @berliner to remove the unstable chrome from the `debian-snap-base` image

Reply:
I'm happy to remove the unstable Chrome. we automatically download stable channel when building image.

Reply:
Chrome is in the base image because that's how we originally built the snap services. It's no longer needed, so it can be removed from the base image altogether. Since it's still a *packaged* Chrome and not a puppeteered one, X library depends will still get added and stuff should not break.

Reply:
@rupl, i have updated the base image. re-review please :)